logging/logging_test.go

```diff
@@ -243,6 +292,50 @@ func TestLogAndEntries(t *testing.T) {
 	}
 }
 
+func TestLogWithMinimalLoggedSeverity(t *testing.T) {
```

Comment:
nit: same comment. the current implementation increases technical debt for tests instead of decreasing it.